Services/AzineTwigSwiftMailer.php

```diff
@@ -518,23 +518,24 @@ private function getMailer($params){
 
     /**
      * Send confirmation link to specified new user email.
-     * @param RecipientInterface $user
+     * @param UserInterface $user
      * @param $confirmationUrl
-     * @param $templateName
      * @param $toEmail
      * @return bool
+     * @deprecated As soon as the "confirm changed email"-feature is merged into the FosUserBundle, this function will be removed again. See https://github.com/azine/FOSUserBundle
      */
-    public function sendUpdateEmailConfirmation(RecipientInterface $user, $confirmationUrl, $templateName, $toEmail)
+    public function sendUpdateEmailConfirmation(UserInterface $user, $confirmationUrl, $toEmail)
```

Comment:
As noted in the php-doc comment, this function is supposed to go away as soon as the feature is available in the FosUserBundle. => So in this commit (that is supposed to be used with the updated FosUserBundle) this function should not exist.